### ISSUE TITLE
Drawer does not appear

### ISSUE THREAD

**Issue by clem983** [manumaticx/Ti.DrawerLayout#61]
Hi,

I'm a beginner with Titanium, and I'm trying to install your drawer module (in a classic project). When I put the example code in my project, a blue action bar appears and the click in the left corner is detected...but nothing else happens and there isn't any error in the log. Does I forgot something to do ?
I have another problem : I change the color in the file colors.xml, but the action-bar is still blue. My SDK version is 5.2.2. I also tried with 5.1.2.
I don't know what to do, can you help me please ?

Thanks a lot :)

**Comment by manumaticx** [comment]
Hi @clem983 
Have you installed the latest version 1.4.0? Are you using the example code from readme?
I have to admit that the example code may not be up-to-date. But the snippet in the readme should work just fine. As well as the demo app (that's an Alloy project).

**Comment by clem983** [comment]
Yes, it's the latest version and the code is from the readme.
I tried to install it in a blank project and it doesn't work... I don't know what I'm doing wrong !
Here is my project's structure : 
- modules 
-- android
--- com.tripvi.drawerlayout
----1.4.0
----- documentation ->index.html
----- libs -> 3 other folders
----- platform -> android - res - layout, etc
----- timodule.xml
----- manifest
- resources
-- android
- changelog, tiapp.xml, etc

I put this in tiapp.xml : 
<android xmlns:android="http://schemas.android.com/apk/res/android">
    	<manifest>
            <application android:theme="@style/Theme.AppCompat.Light.NoActionBar"/>
        </manifest>
    </android>

And this is exactly the same problem, even for the action bar's color ... :/

**Comment by manumaticx** [comment]
Oh! I reproduced it. There's on property missing in that example ( `leftDrawerWidth: 240` ). I'll fix this asap but in the meantime you can workaround this by just passing that property. 
```js
// create the Drawer
var drawer = TiDrawerLayout.createDrawer({
    leftView: leftView,
    centerView: centerView,
    leftDrawerWidth: 240
});
```
Thanks for the report!

**Comment by manumaticx** [comment]
All right. It's fixed in 1.4.1 👍

**Comment by clem983** [comment]
It works ! Thanks a lot ! :)
